src/api/app/services/diff_parser.rb

```diff
@@ -9,7 +9,7 @@ def initialize(content: [])
   end
 
   def call
-    @content.each_line.with_index do |line, line_index|
+    @content&.each_line&.with_index do |line, line_index|
```

Comment:
Hm, one should be enough no? How can this be nil if it's required during `initialize`?

Reply:
I received a message from RuboCop: `Lint/SafeNavigationChain: Do not chain ordinary method call after safe navigation operator.`. This was the automatic fix.

Maybe it is not only `nil`...

Reply:
Maybe? We should know no?

Reply:
Providing the default value in the constructor we do not need to worry about this anymore

Reply:
We didn't take in account `content` being `nil`. This is now fixed, and a test case added. Please have a look again.